NOTICE

```diff
@@ -1,4 +1,4 @@
-Copyright 2019 IOHK
+Copyright 2023 IOHK
```

Comment:
Shouldn't it be 2019-2023?

Reply:
thanks for prodding on this, I'm not going to merge this until I get an answer from our legal team.

Reply:
@lehins on second thought (in case this takes a while to get an answer for), I would like to remove the obviously wrong appendix from the `LICENSE` and leave the `NOTICE` untouched. (and follow up with another PR if/when I get an answer.) yea?

Reply:
Yeah, that sounds great.